File: icore/src/main/java/com/ddu/icore/rx/activityresult/ActivityResultInfo.java
package com.ddu.icore.rx.activityresult;

import android.content.Intent;

/**
 * Created by yzbzz on 2018/1/5.
 */

public class ActivityResultInfo {

    private int requestCode;
    private int resultCode;
    private Intent data;

    public int getRequestCode() {
        return requestCode;
    }

    public void setRequestCode(int requestCode) {
        this.requestCode = requestCode;
    }

    public int getResultCode() {
        return resultCode;
    }

    public void setResultCode(int resultCode) {
        this.resultCode = resultCode;
    }

    public Intent getData() {
        return data;
    }

    public void setData(Intent data) {
        this.data = data;
    }
}

File: icore/src/main/java/com/ddu/icore/rx/activityresult/ActivityResultInfo.kt
package com.ddu.icore.rx.activityresult

import android.app.Activity
import android.content.Intent

/**
 * Created by yzbzz on 2018/1/5.
 */

data class ActivityResultInfo(var requestCode: Int = 0, var resultCode: Int = 0, var data: Intent? = null) {

    val isOk = (resultCode == Activity.RESULT_OK)

    val isCancel = (resultCode == Activity.RESULT_CANCELED)
}
